### ISSUE TITLE
Make AndroidTV home screen look like Web app in « look and feel update »

### ISSUE THREAD

**Issue by dannycool87** [jellyfin/jellyfin-androidtv#552]
Hi!

I was wondering if you can make the home page look like the web and iOS one for the 0.12 update?

The actual one need some love in my opinion:

- The layout and size of the items displayed

- The choice of the poster vs landscape image to look like the web app

-  I was wondering if it will be possible to deactivate the "cardview" to only see the poster and the title without "box" to look like the web app?

- Disable left gray menu (issue 532) and remplace it by the same header of the web and iOS app

In short, to make the application look like the one on the web and on iOS, because, first, it is very beautiful, and second, for the sake of cohesion between the different platforms.  All of this, to the extent possible, of course.

Thank you very much for the work done, you are doing well!

And, Sorry for my English...

**Comment by Dubwize** [comment]
I also believe the on load focus should be the main screen not the pop out menu bar on the left.

Within our individual libraries we should be able to toggle the view to display either all our TV/Movie items (same as the web version), or the Latest / next up section (as it currently has now)

**Comment by nielsvanvelzen** [comment]
Improving the user experience is an ongoing effort but we're mainly focusing on maintainability of the code right now. See our details page rewrite (#269) or the login redesign (#578) as some examples. We're a small team though so it takes a lot of time to make those changes. 

The changes we make might be inspired by the website (although more likely to use jellyfin-vue as inspiration) but most of the time we use our own judgement since we're making an app for televisions, not desktops.

**Comment by thornbill** [comment]
@Dubwize there is already a setting to control the default view per library, but it's a bit hidden. You need to go to the all items view ad select the gear icon at the top.

**Comment by Dubwize** [comment]
@nielsvanvelzen - I will take a look at those thank you.

@thornbill - Thanks I will go and try and find that setting now.  I'm installing a firestick + jellyfin for the inlaws, and I want to make it as easy as possible for them.

**Comment by jcdick1** [comment]
"The choice of the poster vs landscape image to look like the web app"

I just installed a Fire TV gen 3 dongle, and so was my first experience with the Android TV app.  What immediately stood out to me on the initial homepage was that movies used vertical images and TV used horizontal images, within the same row, and it certainly seems awkward.  I was curious if this was a design decision, or if that's just how the client currently receives the data from the server.

**Comment by nielsvanvelzen** [comment]
The sizing of the so-called "cards" everywhere in the app are something we haven't changed yet, I agree that all cards in the same row should have the same height.

**Comment by grravity** [comment]
Curiously, are there any plans to polish up the Android app as well? Or just Android TV?

I think both could use a bit of a overall polishing in the long run. 

Using Kodi on TV for Jellyfin is much better experience due to the extreme customizability overall though and I have no plans on switching from that back to Jellyfin Android TV. Not a fan of using Kodi on mobile however. Was just curious if there are any plans.

**Comment by nielsvanvelzen** [comment]
The mobile app is in it's own repository so questions for that shouldn't be asked here.
But to answer your question; jellyfin-android uses jellyfin-web (for now), so design changes should be done there.

**Comment by nielsvanvelzen** [comment]
Redesigning the app is an ongoing effort and there is not specific way to consider this issue as "done" so I'll close it now. This doesn't mean the current design is final and we'll keep working on it.
